test/test-accessor.rb

```diff
@@ -21,6 +21,7 @@ def setup
     setup_database
     @posts = Groonga::Hash.create(:name => "Posts", :key_type => "ShortText")
     @id = @posts.column("_id")
+    @key = @posts.column("_key")
```

Comment:
We should reduce scope. `@key` is used only `#key_accessor?` test case. It should be defined there not here.